site/composer.json

```diff
@@ -29,6 +29,7 @@
     "php-ds/php-ds": "1.3.0",
     "ramsey/uuid": "4.2.1",
     "spatie/commonmark-highlighter": "2.1.1",
+    "symfony/cache": "^4.4",
```

Comment:
You a set value, in this case `4.4.30`.

Because Submitty is an application that's never imported we don't really gain anything by defining a range of allowed versions of a dependency, versus having a set value makes it clear what version of a dependency we're using (and in the case of symfony that all versions are the same).